test/Libraries/RevitNodesTests/Elements/ElementTests.cs

```diff
@@ -260,6 +260,50 @@ public void ElementCurveReferences_ExtractsExpectedReferences()
 
         #endregion
 
+        #region Pin settings
+        private static void GetExpectedElementPinStatus(Element element, bool value)
+        {
+            bool pinStatus = element.IsPinned;
+
+            Assert.NotNull(pinStatus);
+            Assert.IsTrue(value);
+        }
+        /// <summary>
+        /// Checks if IsPinned returns the correct value
+        /// </summary>
+        [Test]
+        [TestModel(@".\element.rvt")]
+        public void CanSuccessfullyGetElementPinnedStatus()
+        {
+            // gets the pinned  an element from the model
+            // and checks if IsPinned is the correct value
+            var pinnedElement = ElementSelector.ByElementId(184176, true);
+            GetExpectedElementPinStatus(pinnedElement, true);
+
+            var unPinnedElement = ElementSelector.ByElementId(184324, true);
+            GetExpectedElementPinStatus(unPinnedElement, false);
+        }
+
+        private static void SetElementPinnedStatus(Element element)
+        {
+            // check if elements Pinned status can be set as both true and false
+            element.SetPinnedStatus(true);
+            Assert.IsTrue(element.IsPinned);
+
+            element.SetPinnedStatus(false);
+            Assert.IsFalse(element.IsPinned);
+        }
```

Comment:
don't think we need this as a separate method as it's only called once, should be in `CanSuccessfullySetElementPinnedStatus`